### PR TITLE
Update style of tab bar navigation component

### PR DESCRIPTION
**Pull request by bzf**
screenshot 2016-10-18 13 39 52

### CODE REVIEW

**Comment by bzf** [comment]
Regarding the hound comments, it so happens that you have to be pretty specific when overriding styles from inside bootstrap to make them stick. If anyone has any suggestions to avoid this that'd be great 😄

**Comment by jorkas** [comment]
Really nice work, maybe we can display the item stacked and with full width on smaller screens @Joelhelin @bzf?

Problem on smaller screens:
navs-small-screen

The contrast of the color is too low, and so says the [contrast checker](http://webaim.org/resources/contrastchecker/?fcolor=999999&bcolor=ffffff) too. 
contrast

File: src/mnd-bootstrap/components/_navs.scss [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+---
+title: Navs
+name: navs
+category: Components
+---
+
+```html_example_table
+<ul class="nav nav-tabs">
+  <li role="presentation" class="active"><a href="#">Home</a></li>
+  <li role="presentation"><a href="#">Profile</a></li>
```
Comment (jorkas) [inline src/mnd-bootstrap/components/_navs.scss]: Maybe change the links to `href="#navs"` so you don't leave the section when clicking around and checking the state.

**Comment by bzf** [comment]
@jorkas @Joelhelin Should we handle the supertiny case? I don't that will occur anywhere inside the app so I think we could skip it. What do you guys think?

**Pull request merged**